Create ui-c8y-1023-16-3-ui-enable-data-point-merging-on-the-same-axis-by-default.md (#4014)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-16-3-ui-enable-data-point-merging-on-the-same-axis-by-default.md

---
date: ""

> title: New data point graph and data explorer by default merge data points on the same axis

Reviewer: ```suggestion
title: New data point graph and data explorer by default merge data points with the same minimum and maximum values
```

> title: Enable data point merging on the same axis for the new data point graph and data explorer by default

Reviewer: ```suggestion
title: New data point graph and data explorer by default merge data points on the same axis
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65314
version: 1023.16.3
---

> Previously, data points were not merged on the same axis by default in the new data point graph and data explorer, which made it difficult to compare data. With this change, data points are now merged on the same axis by default. This improves the usability and allows users to easily compare and analyze data on the same chart without additional configuration.

Reviewer 2: ```suggestion
Previously, data points with the same minimum and maximum values were not merged on the same axis by default in the new data point graph and Data Explorer, which made comparison more difficult. With this change, the “merge matching data points” option is now enabled by default, so data points with matching minimum and maximum values are automatically displayed on the same axis. This improves usability and allows users to more easily compare and analyze related data on the same chart without requiring additional configuration.
```

> Previously, data points were not merged on the same axis for the new data point graph and data explorer by default which made it difficult to compare data. With this change, data points are now merged on the same axis by default. This improves the usability and allows users to easily compare and analyze data on the same chart without additional configuration.

Reviewer: ```suggestion
Previously, data points were not merged on the same axis by default for the new data point graph and data explorer, which made it difficult to compare data. With this change, data points are now merged on the same axis by default. This improves the usability and allows users to easily compare and analyze data on the same chart without additional configuration.
```

> Previously, data points were not merged on the same axis by default in the new data point graph and data explorer, which made it difficult to compare data. With this change, data points are now merged on the same axis by default. This improves the usability and allows users to easily compare and analyze data on the same chart without additional configuration.

Reviewer: @eniosultan which data points should be merged on the same axis?

Reviewer 2: changed

> Previously, data points were not merged on the same axis by default for the new data point graph and data explorer, which made it difficult to compare data. With this change, data points are now merged on the same axis by default. This improves the usability and allows users to easily compare and analyze data on the same chart without additional configuration.

Reviewer: ```suggestion
Previously, data points were not merged on the same axis by default in the new data point graph and data explorer, which made it difficult to compare data. With this change, data points are now merged on the same axis by default. This improves the usability and allows users to easily compare and analyze data on the same chart without additional configuration.
```

> Previously, data points with the same minimum and maximum values were not merged on the same axis by default in the new data point graph and Data Explorer, which made comparison more difficult. With this change, the “merge matching data points” option is now enabled by default, so data points with matching minimum and maximum values are automatically displayed on the same axis. This improves usability and allows users to more easily compare and analyze related data on the same chart without requiring additional configuration.

Reviewer: ```suggestion
Previously, data points with the same minimum and maximum values were not merged on the same axis by default in the new data point graph and data explorer, which made comparison more difficult. With this change, the “Merge matching data points” option is now enabled by default, so data points with matching minimum and maximum values are automatically displayed on the same axis. This improves usability and allows users to more easily compare and analyze related data on the same chart without requiring additional configuration.
```
